tests/functional/feature_flags/test_schema_validation.py

```diff
@@ -678,23 +678,16 @@ def test_validate_time_condition_between_datetime_range_invalid_condition_value_
         ConditionsValidator.validate_condition_value(condition=condition, rule_name=rule_name)
 
 
-@pytest.mark.parametrize(
-    "cond_value",
-    [
-        {TimeValues.END.value: "10:10", TimeValues.START.value: "2022-10-10T12:15:00Z"},
-        {TimeValues.END.value: "2022-10-10", TimeValues.START.value: "2022-10-10T12:15:00Z"},
```

Comment:
Note: `2022-10-10` is actually a correct ISO8601 format parsed by Python, corresponding to `2022-10-10T00:00:00Z`.